src/game_launcher.cpp

```diff
@@ -210,6 +210,11 @@ game_launcher::game_launcher(const commandline_options& cmdline_opts, const char
 		no_music = true;
 	if (cmdline_opts_.nosound)
 		no_sound = true;
+	if (cmdline_opts_.noupdatecheck) {
```

Comment:
I'm fairly sure this is not the right way to add this setting? If I understand correctly, putting it in preferences means it'll be stored in the preferences file. That means that running once with `--no-update-check` permanently disables update checking unless you edit or delete your preferences file.

Reply:
It will disable update checking until next start. There is also `else` statement here.

Reply:
You are right. It don't belong to preferences. I need to save variable somewhere so I decided to save information about if we will check for updates or not into preferences, because it is acessible from whole code. I think that best way is save it to preferences because in future we can add physical option into preferences dialog to forbid update checking at all. As of now the preferences dialog won't contain any checkbox or something related to update checking.